@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStatus.vue

```diff
@@ -4,9 +4,21 @@
     <NoDataError v-if="hasError" />
     <UiCardSpinner v-else-if="!isReady" />
     <template v-else>
-      <PoolDashboardStatusItem :active="activeHostsCount" :label="$t('hosts')" :total="totalHostsCount" />
+      <PoolDashboardStatusItem
+        :active="activeHostsCount"
+        :label="$t('hosts')"
+        :active-label="$t('host.active', 2)"
+        :inactive-label="$t('host.inactive', 2)"
```

Comment:
Is there a reason to not use the real value?

```suggestion
        :active-label="$t('host.active', activeHostsCount)"
        :inactive-label="$t('host.inactive', activeHostsCount)"
```